RELEASE/scripts/autoscend/auto_equipment.ash

```diff
@@ -175,47 +175,46 @@ boolean tryAddItemToMaximize(slot s, item it)
 
 string defaultMaximizeStatement()
 {
-	string res = "5item,meat";
+	if(auto_my_path() == "Pocket Familiars")
```

Comment:
when #597 is committed should first rebase and afterwards change this to in_pokefam()

Reply:
done